Jarvis-Platform/jarvis-studio
# Hotfix/Clear persisted store data on sign in

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Clear persisted store data on sign in to avoid roles issues.


## Todos
- [x] Clear data when user is in sign-in page


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout hotfix/vuex-persist
```

1. Log in with a super admin user and go to a configurations listing => `Archived` status should be present.
2. Log in with a non admin or super admin account => `Archived` status should not be present and `vuex` data persisted in `localStorage` should be cleared.


## Impacted Areas in Application
- `src/views/auth/SignIn.vue`
